htdocs/mrp/js/lib_dispatch.js.php

```diff
@@ -153,10 +156,12 @@ function addDispatchLine(index, type, mode)
  */
 function addDispatchTR(qtyOrdered, qtyDispatched, index, nbrTrs, warehouseId, inputId, type, qty, mode, $row) {
 	if (qtyOrdered <= 1) {
-		window.alert("Quantity can't be split");
+        let errormsg = "<?php echo $langs->trans('QtyCantBeSplit') ?>";
```

Comment:
For javascript code, use simple quote.
`let errormsg = '...'`
and around the $langs->trans('QtyCantBeSplit'), use   
`dol_escape_js($langs->trans('QtyCantBeSplit'));`